app/scripts/modules/core/application/listExtractor/listExtractor.service.js

```diff
@@ -8,9 +8,12 @@ module.exports = angular
   ])
   .factory('appListExtractorService', function(_) {
 
-    let getRegions = (appList) => {
+    let defaultAccountFilter = (/*cluster*/) => true;
+
+    let getRegions = (appList, accountFilter = defaultAccountFilter) => {
       return _(appList)
```

Comment:
Okay, I see it's really a list of applications. You should still be able to omit the `cluster` part, and go straight to the `serverGroup`s.